docs/configuring.md

```diff
@@ -962,6 +962,18 @@ to know available options.
 
 The CLI option can be passed several times to add several patterns.
 
+Each pattern can be templated using Jinja.
+
+!!! example
+
+    Templating `exclude` patterns using `_copier_operation` allows to have files
+    that are rendered once during `copy`, but are never updated:
+
+    ```yaml
+    _exclude:
+        - "{% if _copier_operation == 'update' -%}src/*_example.py{% endif %}"
+    ```
```

Comment:
I think this deserves an extra explanation:

```suggestion
    ```

    The difference with [skip_if_exists][] is that it will never be rendered during
    an update, no matter if it exitsts or not.
```